src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+.calendar {
+  @each $name, $number in $week-day {
+    &--start-day--#{name} {
+      margin-left: (($number - 1) * $width-square) + ($number - 1);
```

Comment:
The variable `$width-square` is used here. Make sure it is defined and accessible in this context, otherwise, this will cause a compilation error.